website/content/docs/auth/ldap.mdx

```diff
@@ -133,6 +133,10 @@ There are two alternate methods of resolving the user object used to authenticat
 
 @include 'ldap-auth-userfilter-warning.mdx'
 
+#### Alias Dereferencing
+
+- `dereference_aliases` (string, optional) - Control how aliases are dereferenced when performing the search. Possible values are: `never`, `finding`, `searching`, and `always`. `finding` will only dereference aliases during name resolution of the base. `searching` will dereference aliases after name resolution.
+
```

Comment:
Would it be possible to also add the new parameter to the [api-docs](https://github.com/f4z3r/vault/blob/feat/6128/website/content/api-docs/auth/ldap.mdx?plain=1#L93)?

Something like:
```
- `dereference_aliases` `(string: never)` - When aliases should be dereferenced on search operations. Accepted values are 'never', 'finding', 'searching', 'always'. Defaults to 'never'.
```

Reply:
Sure thing. Added your proposed text in the API docs.